ui-components/src/helpers/filters.ts

```diff
@@ -92,7 +92,7 @@ export function decodeFiltersFromFrontendUrl(
   const filterState: ListingFilterState = {}
   let foundFilterKey = false
   for (const queryKey in query) {
-    if (queryKey in FrontendListingFilterStateKeys) {
+    if (queryKey in FrontendListingFilterStateKeys && query[queryKey]) {
```

Comment:
I think this would prohibit using filters like "not senior housing", which we'd like to be able to do.

The purpose of deleting `data[FrontendListingFilterStateKeys.includeNulls]` in `onSubmit` was to avoid this issue, but I guess it's not WAI? Do you know how it's getting passed as an empty string?

Reply:
Ah of course, I forgot about that case. I don't know how it was getting passed as an empty string; there was also once I was seeing it as an empty array. The deletion wasn't working because the form was passing the value as an empty array, which is truthy and thus it wouldn't be deleted. Then the param would stick around.

Reply:
I changed to check for the empty string here; if you think we can do something about the empty array case, too, let me know.

Reply:
We could also explicitly check for "true" and only allow that when the key is `includeNulls`

Reply:
I put in this as a minimum fix for now.